content/en/docs/rate-limits.md

```diff
@@ -253,6 +252,27 @@ We do **not** offer overrides for this limit.
 
 </div>
 
+# Overall Requests Limit
+
+In addition to our [account registration](#account-registration-limits) and
+[certificate issuance](#certificate-issuance-limits) limits, there are
+per-endpoint overall request limits that apply per-IP address. These are
+enforced by our load balancers and are designed to protect the ACME API from
+being overwhelmed by clients that make too many requests at once.
+
+Endpoint             | Requests per IP  | Burst Capacity
+|--------------------|------------------|-----------------|
```

Comment:
```suggestion
|Endpoint            | Requests per IP  | Burst Capacity  |
|--------------------|------------------|-----------------|
```

Markdown tables usually have `|` on each end of the table. Seems like it's not required here but I think we should keep them for consistency